azure_arc_symphony_jumpstart/Bootstrap.ps1

```diff
@@ -0,0 +1,104 @@
+param (
+    [string]$spnClientId,
+    [string]$spnClientSecret,
+    [string]$spnTenantId,
+    [string]$subscriptionId,
+    [string]$resourceGroup,
+    [string]$azureLocation,
+    [string]$clusterName,
+    [string]$extensionName,
+    [string]$customLocationName,
+    [string]$extensionVersion
+)
+
+[System.Environment]::SetEnvironmentVariable('spnClientID', $spnClientId,[System.EnvironmentVariableTarget]::Machine)
```

Comment:
Do we need to save all inputs in machine level environment variable? What is the purpose?